src/app/pages/storage/volumes/zvol/zvol-form/zvol-form.component.ts

```diff
@@ -75,7 +75,21 @@ export class ZvolFormComponent {
     'M': 1048576,
     'K': 1024,
   };
-
+  protected reverseRecordSizeMap: Object= {
```

Comment:
volblocksize and recordsize are distinct things, why are you naming it recordsize? Thats confusing IMO.

Reply:
@william-gr Thanks!, now I am using less confusing variable name.